presto-main/src/main/java/com/facebook/presto/metadata/FunctionAndTypeManager.java

```diff
@@ -253,6 +253,11 @@ public void registerBuiltInFunctions(List<? extends SqlFunction> functions)
                 .collect(toImmutableList());
     }
 
+    public Collection<SqlFunction> listBuiltInFunctions()
```

Comment:
Related: if you are introducing this, maybe worth rewrite `listFunctions` to use this as well. But this doesn't need to be in this diff / PR.

Reply:
done